homeassistant/components/snapcast/media_player.py

```diff
@@ -149,16 +150,27 @@ class SnapcastGroupDevice(MediaPlayerEntity):
 
     def __init__(self, group, uid_part):
         """Initialize the Snapcast group device."""
+        self._attr_available = True
```

Comment:
Why is this initialized to available instead of to unavailable?
(I'm not saying it's wrong, I'm just curious)

Reply:
When the server provides the list of clients/groups, these are automatically available. The control happens always through the server, so even if a client is turned off, it still can be controlled and should be considered as available (state=standby). So the only case where entities are disabled is when the server is disconnected. As an alternative the availablity could be set in `async_added_to_hass()`.
I hope that makes sense.